tests/postgres_tests/models.py

```diff
@@ -1,39 +1,43 @@
-from django.contrib.postgres.fields import (
-    ArrayField, BigIntegerRangeField, DateRangeField, DateTimeRangeField,
-    FloatRangeField, HStoreField, IntegerRangeField,
-)
 from django.db import connection, models
 
+from .fields import *  # NOQA
```

Comment:
do we need import *? It seems like the same names can be imported regardless of vendor, but maybe I missed something (in which case an explanatory comment would be helpful)